OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/LearnDropDownAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package org.edx.mobile.view.adapters
+
+import android.content.Context
+import android.view.View
+import android.widget.AdapterView
+import androidx.core.content.res.ResourcesCompat
+import org.edx.mobile.R
+import org.edx.mobile.databinding.LearnSelectionItemBinding
+
+
+class LearnDropDownAdapter(
+    context: Context,
+    layoutRes: Int
+) : BaseListAdapter<String>(context, layoutRes, null) {
+
+    override fun render(tag: BaseViewHolder?, model: String?) {
+        val holder = tag as LearnItemViewHolder
+        holder.binding.tvLearnItem.text = model
+        var typeface = ResourcesCompat.getFont(context, R.font.inter_regular)
+        if (isSelected(holder.position)) {
+            typeface = ResourcesCompat.getFont(context, R.font.inter_semi_bold)
+        }
+        holder.binding.tvLearnItem.typeface = typeface
+    }
+
+    override fun getTag(convertView: View): BaseViewHolder {
+        val binding = LearnSelectionItemBinding.bind(convertView)
+        return LearnItemViewHolder(binding)
+    }
+
+    private class LearnItemViewHolder(val binding: LearnSelectionItemBinding) :
+        BaseViewHolder()
```

Comment:
Please move this to the end of the class.

Reply:
done.